app/templates/base.html

```diff
@@ -1,127 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>{{ config.APP_NAME }}</title>
+    <!-- meta -->
+    <meta name="description" content="" />
+    <meta name="author" content="" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    {% block meta %}{% endblock %}
+    <!-- styles -->
+    <link
+      href="{{ url_for('static', filename='css/main.css') }}"
+      rel="stylesheet"
+      media="screen"
+    />
+    {% block links %}{% endblock %}
+  </head>
 
-<head>
-  <meta charset="utf-8">
-  <title>{{ config.APP_NAME }}</title>
-  <!-- meta -->
-  <meta name="description" content="">
-  <meta name="author" content="">
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  {% block meta %}{% endblock %}
-  <!-- styles -->
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
-    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-  <link href="{{ url_for('static', filename='css/main.css') }}" rel="stylesheet" media="screen">
-  {% block links %}{% endblock %}
-</head>
+  <body class="relative h-screen bg-blueGray-50">
+    <!-- Header -->
+    {% include 'header.html' %}
+    <!-- Flash Messages -->
+    {% with messages = get_flashed_messages(with_categories=true) %} {% if
+    messages %} {% for category, message in messages %}
```

Comment:
There are cases, when if you move jinja logic to another row, it doesn't work. Not a best option

Reply:
added prettier-ignore